OSSM-3988: [DOC] Update cert-manager procedure intro for installation and verification (#61528)
openshift/openshift-docs: modules/ossm-cert-manager-installation.adoc

[id="ossm-cert-manager-installation_{context}"]
= Installing cert-manager

> You can install `cert-manager` to manage the lifecycle of TLS certificates; ensuring they are valid and up-to-date. If you are running Istio in your environment, you can also install `istio-csr`, a certificate authority (CA) server that handles certificate signing requests (CSR) from Istio proxies. The `istio-csr` CA delegates signing to cert-manager and cert-manager delegates to the configured CA.

Reviewer: Per https://61528--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-security.html#ossm-cert-manager-integration-istio_ossm-security I'd make this consistent and instead go with "cert-manager tool" or "`cert-manager` tool" throughout the docs

> You can install `cert-manager` to manage the lifecycle of TLS certificates; ensuring they are valid and up-to-date. If you are running Istio in your environment, you can also install `istio-csr`, a certificate authority (CA) server that handles certificate signing requests (CSR) from Istio proxies. The `istio-csr` CA delegates signing to cert-manager and cert-manager delegates to the configured CA.

Reviewer: ```suggestion
You can install the `cert-manager` tool to manage the lifecycle of TLS certificates and ensure that they are valid and up-to-date. If you are running Istio in your environment, you can also install the `istio-csr` certificate authority (CA) server, which handles certificate signing requests (CSR) from Istio proxies. The `istio-csr` CA delegates signing to the `cert-manager` tool, which delegates to the configured CA.
```

suggestions for clarity and minimalism

.Verification

> Use the sample `httpbin` service and `sleep` apps to check mTLS traffic from ingress gateways and verify that cert-manager is installed.

Reviewer: ```suggestion
Use the sample `httpbin` service and `sleep` app to check mTLS traffic from ingress gateways and verify that the cert-manager tool is installed.
```
again, be consistent with cert-manager formatting
